git/foundations_git/introduction_to_git.md

```diff
@@ -48,7 +48,7 @@ This section contains questions for you to check your understanding of this less
 *   <a class="knowledge-check-link" href="#git-local">Does Git work at a local or remote level?</a>
 *   <a class="knowledge-check-link" href="#github-remote">Does GitHub work at a local or remote level?</a>
 *   <a class="knowledge-check-link" href="https://www.youtube.com/watch?v=2ReR1YJrNOM">Why is Git useful for developers?</a>
-*   <a class="knowledge-check-link" href="https://www.youtube.com/watch?v=8oRjP8yj2Wo">Why are Git and GitHub useful for a team of developers?</a>
+*   <a class="knowledge-check-link" href="https://www.youtube.com/watch?v=1h9_cB9mPT8&feature=youtu.be&t=13s">Why are Git and GitHub useful for a team of developers?</a>
```

Comment:
```suggestion
*   <a class="knowledge-check-link" href="https://vimeo.com/41027679">Why are Git and GitHub useful for a team of developers?</a>
```
I found the original video on Vimeo, directly from GitHub. I'm going to suggest using this video instead.

Reply:
The video is a good suggestion, but this link refers to the material studied in the "Assignment" section, point 3. If the coordinator accepts the video, it is necessary to change the link in Assignment too.

Reply:
I linked the original video. The dead link you are replacing, that video, I found it on Vimeo. You are replacing the link with another one from the section, and not to a location in the video that answers the knowledge check.

Reply:
I understand. The video that you are suggesting was replaced in the Assignment section for another one. I copied that link of the video to the knowledge section. 
Yep, we can put your suggestion but we must update the same link in the Assignment section.
In my opinion, I would like to maintain the link because the current video is more dynamic and it has a good explanation. Another reason is that all videos that I've watched until now are on YouTube, and are more comfortable on mobile devices, most users have the YouTube app on their mobile. 
Let's wait for suggestions from a member.

Reply:
I updated the link and the timestamp. What do you think of that? Your link uses the original video and a timestamp that skips the intro. I am linking to the same video, but I updated the timestamp to actually be for the knowledge check.
Does that make sense?

Reply:
The time starts in 13 seconds because I think that the video just shows a presentation of the channel. For that reason, I didn't change the timestamp. I don't know if we can change the timestamp recommended by the community.

Reply:
The purpose of the knowledge check is to give the answer to the question. The timestamp I'm providing skips to the answer in the video.

Reply:
Yes, you're right. Show us the new timestamp suggestion, we could show it to the member.

Reply:
I updated it already. Please check again.

Reply:
Sorry, I see the URL "youtu.be". For that reason, I thought it was another video, my fault. Let's do this.